carbonserver/info.go

```diff
@@ -117,6 +113,14 @@ func (listener *CarbonserverListener) infoHandler(wr http.ResponseWriter, req *h
 				SecondsPerPoint: spp,
 				NumberOfPoints:  nop,
 			})
+			//only one metric is enough - first metric
```

Comment:
ProtoV2 doens't support multiple metrics in response here.

I also think it makes more sense to move this sutff to `case protoV2Format`, just grab the first response from protoV3 structure and convert it to protoV2